### PR TITLE
consumer summary REST API - adding more information.

### PR DESCRIPTION
**Pull request by lokresh88**
- Consumer API routes fix. 
- Extra lag information exposed.
- Tests adjusted.

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

**Comment by yahoocla** [comment]
CLA is valid!

File: conf/routes [hunk 1]
```diff
@@ -58,8 +58,8 @@ GET    /api/status/:c/topicIdentities                       controllers.api.Kafk
 GET    /api/status/clusters                                 controllers.api.KafkaStateCheck.clusters
 GET    /api/status/:c/:t/underReplicatedPartitions          controllers.api.KafkaStateCheck.underReplicatedPartitions(c:String,t:String)
 GET    /api/status/:c/:t/unavailablePartitions              controllers.api.KafkaStateCheck.unavailablePartitions(c:String,t:String)
-GET    /api/status/:cluster/:consumer/:topic/topicSummary   controllers.api.KafkaStateCheck.topicSummaryAction(cluster:String, consumer:String, topic:String, consumerType:String)
-GET    /api/status/:cluster/:consumer/groupSummary          controllers.api.KafkaStateCheck.groupSummaryAction(cluster:String, consumer:String, consumerType:String)
+GET    /api/status/:cluster/:consumer/:topic/:consumerType/topicSummary   controllers.api.KafkaStateCheck.topicSummaryAction(cluster:String, consumer:String, topic:String, consumerType:String)
```
Comment (patelh) [inline conf/routes]: Not sure if we want to change this.  It is backwards incompatible change.  You should just be able to pass in a query parameter for consumerType (e.g. ?consumerType=KF)

Comment (lokresh88) [inline conf/routes]: Sorry, i did n't know that. reverted the routes change.

**Pull request merged**